Quiz.Site/Controllers/Surface/ProfileSurfaceController.cs

```diff
@@ -191,11 +193,7 @@ public async Task<IActionResult> EditProfile(EditProfileViewModel model)
                     TempData["ShowToast"] = true;
                 }
             }
-
-            var profilePage = CurrentPage.AncestorOrSelf<HomePage>().FirstChildOfType(ProfilePage.ModelTypeAlias);
```

Comment:
Same with this one, as we're currently on the profilePage, this variable wasn't used.

Reply:
Brill thanks